.version

```diff
@@ -1 +1 @@
-0.6.0
+0.7.0
```

Comment:
FYI, you and @jack-stanek-hpe are both currently trying to make the same version bumps. See #98.

Reply:
Ah yep, and actually #97 has also bumped the version which is causing conflicts for both of us.